From 85faf353316d0c00e8904f7c14a52af417d545a3 Mon Sep 17 00:00:00 2001
From: Kyu Man Lim <kyuman.lim@lge.com>
Date: Fri, 10 Feb 2017 10:21:33 +0900
Subject: [PATCH] Disable Faux bolding in Qts FreeType FontEngine

When Text is set to be bold, Original font glyph is displayed only.
WebOS don't want to apply bolden effect by freetype engine

Upstream-Status : Inappropriate [webOS specific]
---
 src/platformsupport/fontdatabases/freetype/qfontengine_ft.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/platformsupport/fontdatabases/freetype/qfontengine_ft.cpp b/src/platformsupport/fontdatabases/freetype/qfontengine_ft.cpp
index e132442e37..2140dee8e3 100644
--- a/src/platformsupport/fontdatabases/freetype/qfontengine_ft.cpp
+++ b/src/platformsupport/fontdatabases/freetype/qfontengine_ft.cpp
@@ -830,6 +830,8 @@ bool QFontEngineFT::init(FaceId faceId, bool antialias, GlyphFormat format,
                     embolden = true;
             }
         }
+        // webOS want to use original font glyph only, because "embolden" is possible to make fallback font more bold
+        embolden = false;
         // underline metrics
         line_thickness =  QFixed::fromFixed(FT_MulFix(face->underline_thickness, face->size->metrics.y_scale));
         underline_position = QFixed::fromFixed(-FT_MulFix(face->underline_position, face->size->metrics.y_scale));
